src/utils/provider.ts

```diff
@@ -57,3 +74,19 @@ export async function fileinfo(
     Logger.error(error.message)
   }
 }
+
+export async function isFileValid(
+  url: string | DID,
+  providerUri: string,
+  cancelToken: CancelToken
+): Promise<boolean> {
+  const response = await getFileInfo(url, providerUri, cancelToken)
+
+  if (!response || response.status !== 200 || !response.data) return false
+
+  if (!response.data[0] || !response.data[0].valid) return false
+
+  if (response.data[0].contentLength === '0') return false
```

Comment:
this seems to still produce many false positives. Of the current 9 highest liquidity assets, 4 are being reported as unavailable assets which I highly doubt. If we ship this check in this state the top publishers will come to us and ask why we tell people their assets are not available when in fact they are.

Reply:
Yes, should we exclude the `contentLength` check in this PR?

Reply:
it seems so. Just tested and can confirm all highest liquidity assets are fine with this check removed. Still kinda weird casue upon publishing we seemed to have been able to correctly detect the size on all of them but they were published with our old `fileinfo` Node.js app endpoint. But don't see why `provider` would not be able to get the `contentLength`

Reply:
Yes...the checking gets crazier if you test it with empty string, `contentLength === ""`. Not sure why plenty of endpoint is returning `""`.

Reply:
I had removed the contentLength check for now.

Reply:
can you get some dids ?
I will open an issue on provider to see why do we get such responses

Reply:
@alexcos20  
These are the DID of highest liquidity from ETH mainnet:

DID with `contentLength = ""` 
- did:op:a2B8b3aC4207CFCCbDe4Ac7fa40214fd00A2BA71 
- did:op:fcB47f5781F14Ed7E032BD395113b84C897aA23f

DID with `contentLength = "0"`
- did:op:7Bce67697eD2858d0683c631DdE7Af823b7eea38
- did:op:2a76F680279CE629a9F5E601BDa7246e06F226f0